zhaquirks/thirdreality/switch.py

```diff
@@ -1,118 +1,41 @@
 """Third Reality switch devices."""
 
-from zigpy.profiles import zha
-from zigpy.quirks import CustomDevice
-from zigpy.zcl.clusters.general import Basic, Groups, Identify, OnOff, Ota, Scenes
-from zigpy.zcl.clusters.homeautomation import Diagnostic
-
-from zhaquirks import PowerConfigurationCluster
-from zhaquirks.const import (
-    DEVICE_TYPE,
-    ENDPOINTS,
-    INPUT_CLUSTERS,
-    MODELS_INFO,
-    OUTPUT_CLUSTERS,
-    PROFILE_ID,
+from typing import Final
+
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import QuirkBuilder
+from zigpy.quirks.v2.homeassistant import UnitOfTime
+import zigpy.types as t
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+
+class ThirdRealitySwitchCluster(CustomCluster):
+    """Third Reality's switch private cluster."""
+
+    cluster_id = 0xFF02
+
+    class AttributeDefs(BaseAttributeDefs):
+        """Define the attributes of a private cluster."""
+
+        count_down_time: Final = ZCLAttributeDef(
+            id=0x0001,
+            type=t.uint16_t,
+            is_manufacturer_specific=True,
+        )
+
+
+(
+    QuirkBuilder("Third Reality, Inc", "3RSS009Z")
+    .replaces(ThirdRealitySwitchCluster)
+    .number(
+        attribute_name=ThirdRealitySwitchCluster.AttributeDefs.count_down_time.name,
+        min_value=0,
+        max_value=65535,
+        step=1,
+        unit=UnitOfTime.SECONDS,
+        cluster_id=ThirdRealitySwitchCluster.cluster_id,
+        translation_key="count_down_time",
+        fallback_name="Count down time",
+    )
+    .add_to_registry()
 )
-from zhaquirks.thirdreality import THIRD_REALITY
-
-
-class CustomPowerConfigurationCluster(PowerConfigurationCluster):
-    """Custom PowerConfigurationCluster."""
-
-    MIN_VOLTS = 2.1
-    MAX_VOLTS = 3.0
```

Comment:
Also, the previous implementation had a `CustomPowerConfigurationCluster` that replaced the percentage reported by the device by one calculated from minimum and maximum voltage.

Is that no longer needed? So the device reports the percentage value itself properly?